elements/lisk-api-client/src/ws_channel.ts

```diff
@@ -165,10 +134,33 @@ export class WSChannel {
 	}
 
 	public subscribe<T = Record<string, unknown>>(eventName: string, cb: EventCallback<T>): void {
+		const request = {
+			jsonrpc: '2.0',
+			id: this._requestCounter,
+			method: 'subscribe',
+			params: {
+				topics: [eventName],
```

Comment:
topics is an array, is it possible to subscribe to multiple events at once? if not, then maybe we can keep it singular topic as a string?

Reply:
yes, topics is array, and in theory it's possible to subscribe multiple at once. right now it's just not allowing per interface, but maybe we could improve it later?